src/Symfony/Component/ErrorHandler/BufferingLogger.php

```diff
@@ -18,7 +18,7 @@
  *
  * @author Nicolas Grekas <p@tchwork.com>
  */
-class BufferingLogger extends AbstractLogger
+final class BufferingLogger extends AbstractLogger
```

Comment:
you still haven't marked it as `@internal`

Reply:
I changed my mind after a talk with @nicolas-grekas